services/ui/ws/window_tree.cc

```diff
@@ -2465,6 +2466,14 @@ void WindowTree::PerformWindowMove(uint32_t change_id,
     return;
   }
 
+  if (window_server_->IsInExternalWindowMode()) {
```

Comment:
maybe you could move this if above line 2444?

if (success && window_server_->IsInExternalWindowMode()) {
  // comment
  // todo
  OnChangeComplete(change_id, true);
  return;
}

Reply:
Well, the if (success...) is redundant here. I left it by accident, but if I move it above 2444, then checks on lines 2454 and 2461 are going to be missed, which is not a right way to do.